app/views/admin/terms_of_service_files/show.html.haml

```diff
@@ -14,6 +14,6 @@
       = t(".using_default_terms_html", tos_link: link_to_platform_terms)
 
 = form_for [main_app, :admin, @new_file] do |f|
-  = f.label :attachment
+  = f.label t(".attachment")
```

Comment:
I was wondering why you changed the spec to use the input name instead of this label and then I realised that we broke the connection between label and input field with this change. We can keep the connection and add a label:
```suggestion
  = f.label :attachment, t(".attachment")
```

This has the advantage that the user can click on the label to upload a file.